File: Python/Matemática/2170 - Juros do Projeto.py
"""
Autor: Pierre Vieira
Data da submissão: 24/02/2020 18:54:35
"""
cont = 1
while True:
    try:
        a, b = map(float, input().split())
    except EOFError:
        break
    else:
        print('Projeto {}:'.format(cont))
        print('Percentual dos juros da aplicacao: {:.2f} %\n'.format(100 * ((b - a)/a)))
    cont += 1

File: Kotlin/Matemática/2170 - Juros do Projeto.kt
/*
* Autor: Pierre Vieira
* Data de submissão: 20/03/2023 03:05:34
*/

fun main(args: Array<String>) {
    var projectCounter = 1
    while (true) {
        val (x, y) = readLine()?.split(" ")?.map { it.toDouble() } ?: break
        val jurosPercentual = ((y / x) - 1) * 100
        println("Projeto $projectCounter:")
        val result = String.format("%.2f", jurosPercentual)
        println("Percentual dos juros da aplicacao: $result %\n")
        projectCounter++
    }
}